src/slider/src/styles/index.cssr.ts

```diff
@@ -125,7 +125,7 @@ export default c([
       cursor: not-allowed;
       opacity: var(--n-opacity-disabled);
     `, [
-      cB('slider-handle', `
+      cB('slider-handle-content', `
```

Comment:
rename original `handle` to `handle-wrapper`.
move original handle dom inside `handle-wrapper` and keep it named `handle`.